code/index.html

```diff
@@ -3,28 +3,170 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <meta name="author" content="Ylva Nilsson" />
+    <meta name="description" content="A business site" />
+    <title>Project 3: The busines site</title>
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header>
+      <!-- Navbar menu -->
+      <nav class="navbar-menu">
+        <div class="hamburger-menu">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
 
-    <!-- Signup form -->
+        <img class="logo" src="./icons/icons8-sailing-ship-96.png" alt="logo" />
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+        <div class="row-menu">
+          <a class="home" href="#">HOME</a>
+          <a class="about" href="#">ABOUT</a>
+          <a class="contact" href="#">CONTACT</a>
+          <a class="four" href="#">FOUR</a>
+        </div>
+      </nav>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <!-- Business name and slogan -->
+      <div class="business-name-slogan">
+        <h1>SKEPP OHOJ!</h1>
+        <p>- Join our Saling courses</p>
       </div>
+    </header>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+    <main>
+      <section class="signup-form">
+        <h2>Sign up for a course</h2>
+
+        <form class="form" action="http://httpbin.org/anything" method="POST">
+          <div class="form-group form-group-name">
+            <label for="name">Name</label>
+            <input
+              type="text"
+              name="name"
+              placeholder="First and Last name"
+              required
+            />
+          </div>
+
+          <div class="form-group form-group-email">
+            <label for="email">Email</label>
+            <input
+              type="email"
+              name="email"
+              placeholder="example@email.com"
+              required
+            />
```

Comment:
Nice to see some simple yet effective validation with the 'required' attribute, but you should check how you're using the label and 'for', which should connect with the id of the input. Here's some reading:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label